package.json

```diff
@@ -17,87 +17,89 @@
     "enumify": "^2.0.0",
-    "graphiql": "^0.17.5",
-    "graphql": "^14.5.8",
-    "graphql-tag": "^2.10.1",
-    "jshint": "^2.10.2",
+    "graphiql": "^1.0.3",
+    "graphql": "^15.3.0",
+    "graphql-tag": "^2.11.0",
+    "jshint": "^2.12.0",
     "nprogress": "^0.2.0",
-    "react": "^15.6.2",
-    "react-dom": "^15.6.2",
-    "tippy.js": "^4.3.5",
-    "vue": "^2.6.10",
-    "vue-i18n": "^8.14.1",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
+    "tippy.js": "^6.2.6",
+    "vue": "^2.6.11",
+    "vue-i18n": "^8.21.0",
     "vue-markdown": "^2.2.4",
-    "vue-meta": "^2.3.1",
-    "vue-router": "^3.1.3",
-    "vue-spinner": "^1.0.3",
+    "vue-meta": "^2.4.0",
+    "vue-router": "^3.4.3",
+    "vue-spinner": "^1.0.4",
     "vue-the-mask": "^0.11.1",
     "vuetify": "^2.2.6",
-    "vuex": "^3.1.1",
+    "vuex": "^3.5.1",
     "vuex-router-sync": "^5.0.0"
   },
   "devDependencies": {
-    "@babel/plugin-proposal-class-properties": "^7.8.3",
-    "@cypress/code-coverage": "^2.0.5",
-    "@cypress/webpack-preprocessor": "^5.1.2",
-    "@mdi/font": "^4.5.95",
-    "@vue/cli-plugin-babel": "^3.12.0",
-    "@vue/cli-plugin-e2e-cypress": "^4.3.1",
-    "@vue/cli-plugin-eslint": "^4.2.3",
-    "@vue/cli-plugin-unit-mocha": "^4.2.3",
-    "@vue/cli-service": "^4.2.2",
-    "@vue/test-utils": "^1.0.0-beta.29",
-    "apollo-cache-inmemory": "^1.6.3",
-    "apollo-client": "^2.6.4",
-    "apollo-link": "^1.2.13",
-    "apollo-link-http": "^1.5.16",
-    "apollo-link-ws": "^1.0.19",
-    "axios": "^0.19.0",
-    "babel-eslint": "^10.0.3",
+    "@babel/plugin-proposal-class-properties": "^7.10.4",
+    "@cypress/code-coverage": "^3.8.1",
+    "@cypress/webpack-preprocessor": "^5.4.4",
+    "@mdi/font": "^5.5.55",
+    "@vue/cli-plugin-babel": "^4.5.4",
+    "@vue/cli-plugin-e2e-cypress": "^4.5.4",
+    "@vue/cli-plugin-eslint": "^4.5.4",
+    "@vue/cli-plugin-unit-mocha": "^4.5.4",
+    "@vue/cli-service": "^4.5.4",
+    "@vue/test-utils": "^1.0.4",
+    "apollo-cache-inmemory": "^1.6.6",
+    "apollo-client": "^2.6.10",
+    "apollo-link": "^1.2.14",
+    "apollo-link-http": "^1.5.17",
+    "apollo-link-ws": "^1.0.20",
+    "axios": "^0.19.2",
+    "babel-eslint": "^10.1.0",
     "chai": "^4.2.0",
-    "cross-env": "^6.0.3",
-    "cross-fetch": "^3.0.4",
-    "eslint": "^7.0.0",
+    "cross-env": "^7.0.2",
+    "cross-fetch": "^3.0.5",
+    "eslint": "^7.7.0",
     "eslint-config-standard": "^14.1.1",
     "eslint-config-vuetify": "^0.5.0",
+    "eslint-plugin-import": "^2.22.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-standard": "^4.0.1",
     "eslint-plugin-vue": "^6.2.2",
     "istanbul-instrumenter-loader": "^3.0.1",
     "istanbul-lib-coverage": "^3.0.0",
-    "lodash": "^4.17.15",
-    "null-loader": "^3.0.0",
-    "nyc": "^15.0.1",
-    "sass": "^1.26.3",
-    "sass-loader": "^8.0.2",
-    "sinon": "^7.5.0",
-    "standard": "^14.3.1",
-    "subscriptions-transport-ws": "^0.9.16",
+    "lodash": "^4.17.20",
```

Comment:
I think `lodash` is actually a runtime dependency. Maybe it's working now if another dependency is loading it (the Apollo libraries do). But this needs to be moved to the `dependencies` and removed from the `devDependencies`.